mediacapture-streams/OWNERS

```diff
@@ -1,3 +1,4 @@
 @alvestrand
 @dontcallmedom
 @foolip
+@agouaillard
```

Comment:
@alvestrand : we decided with dom to follow the chapters in the spec for easier spec <-> tests scrum'ing.

@foolip will do.